instrumentation/opentelemetry-instrumentation-asyncio/tests/test_asyncio_future_cancellation.py

```diff
@@ -0,0 +1,59 @@
+import asyncio
+from unittest.mock import patch
+
+import pytest
+
+from opentelemetry.instrumentation.asyncio import AsyncioInstrumentor
+from opentelemetry.instrumentation.asyncio.environment_variables import (
+    OTEL_PYTHON_ASYNCIO_FUTURE_TRACE_ENABLED,
+)
+from opentelemetry.test.test_base import TestBase
+from opentelemetry.trace import get_tracer
+
+
+class TestTraceFuture(TestBase):
+    @patch.dict(
+        "os.environ", {OTEL_PYTHON_ASYNCIO_FUTURE_TRACE_ENABLED: "true"}
+    )
+    def setUp(self):
+        super().setUp()
+        self._tracer = get_tracer(
+            __name__,
+        )
+        self.loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(self.loop)
+        self.instrumentor = AsyncioInstrumentor()
+        self.instrumentor.instrument()
+
+    def tearDown(self):
+        super().tearDown()
+        self.instrumentor.uninstrument()
+        self.loop.close()
+
+    @pytest.mark.asyncio
```

Comment:
Do we have pytest-asyncio installed?

Reply:
Yes, i've seen it being used in a lot of places.

Reply:
I understand, I've made changes based on your feedback.
Thank you.

Reply:
I left a suggestion below